.github/workflows/checks.yaml

```diff
@@ -45,12 +45,24 @@ jobs:
       - name: Test
         run: make test
 
-      - name: Upload coverage report
+      - name: Upload unit test coverage report
         uses: codecov/codecov-action@v3
         with:
           files: ./coverage-unit.out
           flags: unit
 
+      - name: Upload integration test coverage report
```

Comment:
What does it mean when multiple coverage reports are uploaded? Do they all get merged?

Reply:
Yeah, Codecov already merges unit and acceptance code coverage reports. The [report](https://app.codecov.io/gh/hacbs-contract/ec-cli/pull/205) has overall the same percent covered, with the expected drop in unit test coverage